src/main/java/com/example/todolist/API/Restful/Dto/Request/SignInRequest.java

```diff
@@ -0,0 +1,11 @@
+package com.example.todolist.API.Restful.Dto.Request;
+
+import com.example.todolist.Domain.Validation.UserValidation.EmailRule;
+import com.example.todolist.Domain.Validation.UserValidation.PasswordRule;
+
+public record SignInRequest(
+        @EmailRule
+        String email,
+        @PasswordRule
```

Comment:
using Password Rule here against the security standard, just use @NotNull with @NotBlank